### ISSUE TITLE
Improvement of integration GEM with powerplantmatching

### ISSUE THREAD

**Issue by ekatef** [pz-max/pypsa-africa#1080]
Have tested the idea to exclude non-Global Energy Monitor (GEM) data from the sources used by powerplantmatching. To do so, GEO and GPD have been commented out in [configs/powerplantmatching.config.yaml](https://github.com/pypsa-meets-earth/pypsa-earth/blob/main/configs/powerplantmatching_config.yaml).

Testing has been done for Uruguay as an example, and here are the outputs:
1) Having the default set of sources gives in a more complete picture -- GEM-only powerplantmatching (ppm) search returns only less than 1/3 of the power plants found by the default ppm configuration.
2) Interactions between GEM sources and others can have some side effects -- in case of Uruguay the default configuration lead to some mess for Punta Del Tigre power plant. In reality, there are two plants, [the first](https://www.gem.wiki/Punta_del_Tigre_A_power_station) consisting of 6 gas turbine blocks (300 MW in total), [the second one](https://www.gem.wiki/Punta_del_Tigre_B_power_station) being CCGT of 540 MW. If GEM only is used, it results in merging two blocks to a single CCGT of 840 MW, while the default configuration leads to double-counting of GTs as an additional fuelled block of 300MW.

3) Installed wind generation capacity is completely lost from our data workflow for some reason, while in fact wind generates ~1/3 of the electricity. Interestingly, wind installed capacity presents in the online version of GEM.

Basically, it confirms what has been noticed before: adding GEM leads to a considerable decrease of the capacities which are found by ppm for each of the world regions. 

It also appears that interaction of GEM with powerplantmatching can be improved. In particular, some side-effects of mixing GEM data with other data sources can be expected for the generation sites with a multiple power blocks with similar names. That is linked with the fact that ppm heavily relays on the names when making matching of the generation capacities.

Would be also great to capture more data on existing RES capacities, which could be probably resolved by updating or enhancing GEM resources included into ppm

**Comment by ekatef** [comment]
Is linked with #407 and #753

**Comment by ekatef** [comment]
Interestingly, wind power plants are included into the ppm, as it appears from [the specification](https://github.com/PyPSA/powerplantmatching/blob/071748049513c3e37a4bad2746784766466d9c06/powerplantmatching/package_data/config.yaml#L172-L176C8) of the data sources included, there are data for the major wind power plants in Uruguay:
<img width="488" alt="image" src="https://github.com/user-attachments/assets/3c3d3e83-b9d9-44b2-b54b-7acf62205c33">

So, it appears that something goes wrong in matching for GEM, even without interactions with other databases
